HW-5

getlehren opened the pull request.

getlehren committed:
HW-5

dmitry-sviridov commented on src/ru/getman/java/core/AppData.java:
```diff
@@ -0,0 +1,64 @@
+        try (BufferedWriter out = new BufferedWriter(new FileWriter(filename + ".csv", true))) {
+            out.write("\n" + sideOne + ";" + sideTwo + ";" + sideThree);
+        }
+    }
```
Саня, если ты сюда залетишь - не проверяй работу. Эти методы будут перепилены.

getlehren committed:
HW-5
1.Выводит сообщение об ошибке, в том случае, если файл с таким именем  не создан, затем такой файл создается.
2.Подается не отдельное значение , а массив .

dmitry-sviridov commented on src/ru/getman/java/core/AppData.java:
```diff
@@ -0,0 +1,67 @@
+
+    public void addDataToFile(int[] line, String filename) throws IOException {
+        fileLoader(filename);
+        if (headers == null) {
```
вот это зачем?

dmitry-sviridov commented on src/ru/getman/java/core/AppData.java:
```diff
@@ -0,0 +1,67 @@
+        }
+    }
+
+    public void addDataToFile(int[] line, String filename) throws IOException {
```
почему int[] line когда должно быть int[][] lines?
